Sources/HummingbirdAuth/Authenticator/Authenticator.swift

```diff
@@ -55,24 +55,23 @@ public protocol Authenticatable: Sendable {}
 ///     }
 /// }
 /// ```
-public protocol AuthenticatorMiddleware: RouterMiddleware where Context: AuthRequestContext {
+public protocol AuthenticatorMiddleware: RouterMiddleware where Context: AuthRequestContext<Identity> {
     /// type to be authenticated
-    associatedtype Value: Authenticatable
+    associatedtype Identity: Authenticatable
```

Comment:
I'm not sure we need the `Authenticatable` protocol anymore. The identity can be anything as long as it is Sendable.